src/metabase/email/creator_sentiment_email.mustache

```diff
@@ -0,0 +1,34 @@
+{{> metabase/email/_header }}
+  <div class="card-container" style="color: {{colorTextMedium}};">
+    {{{self-hosted}}}
```

Comment:
I think it'd be a little cleaner to have the HTML in here for the self-hosted warning, and just show it conditionally based on a `self-hosted` boolean passed in via the context (rather than building the HTML in Clojure). You can do conditionals like `{{#self-hosted}} content {{/self-hosted}}` — see `pulse.mustache` for some examples